server/src/main/java/io/crate/session/Session.java

```diff
@@ -393,7 +393,7 @@ public void bind(String portalName,
         try {
             preparedStmt = getSafeStmt(statementName);
         } catch (Throwable t) {
-            jobsLogs.logPreExecutionFailure(UUIDs.dirtyUUID(), null, SQLExceptions.messageOf(t), sessionSettings.sessionUser());
+            jobsLogs.logPreExecutionFailure(UUIDs.dirtyUUID(), statementName, SQLExceptions.messageOf(t), sessionSettings.sessionUser());
```

Comment:
Isn't this a bit confusing if the statement name is ending up in the stmt column? I'd tend to use `""` here instead.

Reply:
https://github.com/crate/crate/pull/17499